### DISCUSSION TITLE
404 not found (route not found). Creating and editing: A question about data management.

### DISCUSSION THREAD

**Discussion by demian18** [orchidsoftware/platform#2808] [Q&A]
Hi.
I apologize in advance for such a stupid question, but I have not found a solution.

I followed the documentation, in particular the data management guide. Create a Post model, write two routes for interacting with screens (Edit and Last).Error 404 not found (route not found) occurred when creating the record. Editing and displaying the work.

I found that if I change the $post variable to $posts in the "query" and "CreateOrUpdate" functions in the edit screen file, the error will be fixed. But there will be a problem with editing.

At the moment, I have found a solution only by creating a separate screen and route. I would like to use an example from the documentation (it seems to me that this is the correct and simple option, but if I am wrong, I would be glad of good advice).
Thanks!

**Comment by igorpetrowww** [comment]
Check arguments of `createOrUpdate`. I'm facing similar problem, and the reason was wrong first argument, that I've copied by mistake from my other project

**Reply by demian18** [reply]
Hi @igorpetrowww , i apologize for the long reply and you are absolutely right. My mistake was that I studied the documentation in only one language (In English version found the right answer).
